huaweicloud/provider.go

```diff
@@ -759,6 +759,8 @@ func Provider() *schema.Provider {
 			"huaweicloud_vbs_backup":        resourceVBSBackupV2(),
 			"huaweicloud_vbs_backup_policy": resourceVBSBackupPolicyV2(),
 
+			"huaweicloud_vod_transcoding_template_group": vod.ResourceTranscodingTemplateGroup(),
```

Comment:
please put the resource together with `huaweicloud_vod_media_category`

Reply:
fixed